assets/js/base-components/Form/BaseForm.js

```diff
@@ -271,6 +271,7 @@ const BaseForm = {
         onFail(errors) {
             this.submiting = false;
             var bag = this.$validator.errors;
+            bag.clear();
```

Comment:
why do you need to clear the bag?

Reply:
every time form is validated a new item is pushed to bag array. if i need to display a warning message at the top of form listing all errors using bag items you notice duplicate messages (old error and new error). i see clearing the bag is one way to maintain a only latest errors

Reply:
Sure, thanks. I'm going to merge this.